packages/embedded-wallet-service-rainbowkit/src/lib/wallet.ts

```diff
@@ -22,14 +22,15 @@ export const PaperEmbeddedWalletRainbowKitWallet = <
   T extends RecoveryShareManagement = RecoveryShareManagement.USER_MANAGED,
 >(
   config: PaperEmbeddedWalletRainbowKitWalletProps<T>,
+  supportedChains: Chain[]
```

Comment:
yes it should be optional, i made the supportedChains argument optional everywhere to prevent it from becoming a breaking change, so it will still work if the user does not provide this argument. They will simply be limited to a singular chain.